README.md

```diff
@@ -1,3 +1,5 @@
+https://twiki.cern.ch/twiki/bin/view/CMSPublic/SDTCMSSW_aptinstaller
+
```

Comment:
Right, forgot to remove. The initial idea was to do it on Ubuntu (and therefore travis)